content/docs/user-guide/managing-external-data.md

```diff
@@ -52,15 +52,21 @@ in the same external/remote file.
 Let's take a look at the following operations on all the supported location
 types:
 
-1. Adding a `dvc remote` in the same location as the desired outputs, and
-   configure it as external <abbr>cache</abbr> with `dvc config`.
+1. Configure an external <abbr>cache</abbr> directory (added as a
+   `dvc remote`\*) in the same location as the external data, using
+   `dvc config`.
 2. Tracking existing data on the external location using `dvc add` (`--external`
    option needed). This produces a `.dvc` file with an external URL or path in
    its `outs` field.
 3. Creating a simple stage with `dvc run` (`--external` option needed) that
    moves a local file to the external location. This produces an external output
    in `dvc.yaml`.
 
+> \* Note that for certain `dvc remote` authentication methods, extra config
```

Comment:
`dvc remote` authentication method - sounds like the `dvc remote` command itself needs some auth

Reply:
Fixed.